libs/ledger-live-common/src/families/hedera/tests/js-transaction.test.ts

```diff
@@ -25,7 +26,7 @@ const account: Account = {
   blockHeight: 0,
   currency: {
     type: "CryptoCurrency",
-    id: "",
+    id: "" as CryptoCurrencyId,
```

Comment:
Same here, doesn't seem normal to cast an empty string to a very specific list of currency ids